package.json

```diff
@@ -265,7 +266,8 @@
 					"layer-check",
 					"syncpack:deps",
 					"syncpack:versions",
-					"check:versions"
+					"check:versions",
+					"generate:packagesMd"
```

Comment:
Nit: `checks` suggests to me that there should be no side effects from running it, so I'd be inclined to put this as a direct dependency of `build` (and same for `generate:packageList`). Not sure if something else depends on `checks` and thus would also need to have explicit dependencies on the two `generate:*` ones, but I think I'd still rather see these two somewhere else.

Reply:
I agree in principle that "checks" should have no side effects. However, the downside of doing it in the build is that you learn about the problem really late in the pipeline, since there's no real check, it's just relying on the no extraneous edits check. That said, hopefully people are running local builds before creating PRs, so this may not be an issue in practice. I'll move it to the build phase.